app/assets/stylesheets/application.scss

```diff
@@ -66,6 +73,24 @@
    //  @include vertical-align;
  }
 
+ .ghost-button-thick-border {
+   display: inline-block;
+   width: 200px;
+   padding: 8px;
+   font-weight: bold;
+   color: #fff;
+   border: 1px solid $mustard-yellow;
+   text-align: center;
+   outline: none;
+   text-decoration: none;
+  background-color: transparent;
+ }
+ .ghost-button-thick-border:hover,
+ .ghost-button-thick-border:active {
+   background-color: $mustard-yellow;
+   color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.